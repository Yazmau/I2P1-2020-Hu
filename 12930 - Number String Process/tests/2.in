99
495743616514683422353473195139628365618761316476785989999999978882484442253725857437864833186162671
19
97 98 1
89 93 4
81 85 4
73 77 4
67 70 3
61 64 3
53 57 4
49 51 2
41 45 4
35 38 3
33 34 1
29 31 2
21 25 4
19 20 1
17 18 1
11 14 3
9 10 1
7 8 1
1 4 3
