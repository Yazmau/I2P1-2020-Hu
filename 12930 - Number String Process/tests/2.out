7
5180436
28382574
31463804
356168
649312
99980001
8330
6273372
491436
18
2688
45006003
15
6
160056
5
6
417252
